gulpfile.js

```diff
@@ -61,31 +59,37 @@ gulp.task('scss-lint', function (done) {
 gulp.task('eslint', function (done) {
 
   if (!cFlags.test) {
-    gutil.log('eslint', 'Disabling linting');
+    gutil.log(gutil.colors.cyan('eslint'), 'Disabling linting');
     return done();
   }
 
   return gulp.src('./assets/scripts/**/*.js')
-    .pipe(size())
     .pipe(eslint());
 
 });
 
 gulp.task('styles:homepage', [ 'scss-lint' ], function () {
 
-  gutil.log('styles:homepage', 'Compiling Sass assets');
+  gutil.log(gutil.colors.cyan('styles:homepage'), 'Compiling Sass assets');
 
   var sassStream = sass();
-  var stream = gulp.src('./assets/styles/homepage.scss')
-    .pipe(sassFiles);
+  var stream = gulp.src('./assets/styles/homepage.scss');
 
   if (cFlags.production) {
-    gutil.log('styles', 'Compressing styles');
+    gutil.log(gutil.colors.cyan('styles:homepage'), 'Compressing styles');
     sassStream = sass({ outputStyle: 'compressed' });
   }
 
-  stream = stream.pipe(sassStream.on('error', sass.logError))
-    .pipe(size())
+  stream = stream.pipe(sassStream)
+    .on('error', function (error) {
```

Comment:
This function now formats the error for readability and doesn't rely on the `gulp-sass` plugin.